$MirOS: ports/graphics/tiff/patches/patch-libtiff_Makefile_am,v 1.1 2005/04/24 00:09:11 bsiegert Exp $
--- libtiff/Makefile.am.orig	Tue Jan 11 15:06:44 2005
+++ libtiff/Makefile.am	Sun Apr 24 01:02:33 2005
@@ -107,12 +107,13 @@ endif
 libtiff_la_SOURCES = $(SRCS)
 libtiff_la_LDFLAGS = \
 	-no-undefined \
-	-version-number $(LIBTIFF_MAJOR_VERSION):$(LIBTIFF_MINOR_VERSION):$(LIBTIFF_MICRO_VERSION)
+	-version-number $(LIBTIFF_MAJOR_VERSION)$(LIBTIFF_MINOR_VERSION):$(LIBTIFF_MICRO_VERSION):0
 
 libtiffxx_la_SOURCES = $(SRCSXX)
 libtiffxx_la_LDFLAGS = \
 	-no-undefined \
-	-version-number $(LIBTIFF_MAJOR_VERSION):$(LIBTIFF_MINOR_VERSION):$(LIBTIFF_MICRO_VERSION)
+	-version-number $(LIBTIFF_MAJOR_VERSION)$(LIBTIFF_MINOR_VERSION):$(LIBTIFF_MICRO_VERSION):0
+libtiffxx_la_LIBADD = libtiff.la
 
 #
 # The finite state machine tables used by the G3/G4 decoders
